{
  "dataType": "CVE_RECORD",
  "containers": {
    "adp": [
      {
        "title": "CVE Program Container",
        "references": [
          {
            "url": "https://gitlab.com/daniele_m/cve-list/-/blob/main/README.md",
            "tags": [
              "x_transferred"
            ]
          }
        ],
        "providerMetadata": {
          "orgId": "af854a3a-2127-422b-91ae-364da2661108",
          "shortName": "CVE",
          "dateUpdated": "2024-08-02T22:55:41.263Z"
        }
      },
      {
        "title": "CISA ADP Vulnrichment",
        "metrics": [
          {
            "other": {
              "type": "ssvc",
              "content": {
                "id": "CVE-2023-52289",
                "role": "CISA Coordinator",
                "options": [
                  {
                    "Exploitation": "none"
                  },
                  {
                    "Automatable": "yes"
                  },
                  {
                    "Technical Impact": "partial"
                  }
                ],
                "version": "2.0.3",
                "timestamp": "2024-11-14T14:27:36.464533Z"
              }
            }
          }
        ],
        "providerMetadata": {
          "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
          "shortName": "CISA-ADP",
          "dateUpdated": "2024-11-14T14:29:02.244Z"
        }
      }
    ],
    "cna": {
      "affected": [
        {
          "vendor": "n/a",
          "product": "n/a",
          "versions": [
            {
              "status": "affected",
              "version": "n/a"
            }
          ]
        }
      ],
      "references": [
        {
          "url": "https://gitlab.com/daniele_m/cve-list/-/blob/main/README.md"
        }
      ],
      "descriptions": [
        {
          "lang": "en",
          "value": "An issue was discovered in the flaskcode package through 0.0.8 for Python. An unauthenticated directory traversal, exploitable with a POST request to a /update-resource-data/<file_path> URI (from views.py), allows attackers to write to arbitrary files."
        }
      ],
      "problemTypes": [
        {
          "descriptions": [
            {
              "lang": "en",
              "type": "text",
              "description": "n/a"
            }
          ]
        }
      ],
      "providerMetadata": {
        "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "shortName": "mitre",
        "dateUpdated": "2024-01-13T04:05:25.039527"
      }
    }
  },
  "cveMetadata": {
    "cveId": "CVE-2023-52289",
    "state": "PUBLISHED",
    "dateUpdated": "2024-11-14T14:29:05.946Z",
    "dateReserved": "2023-12-31T00:00:00",
    "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
    "datePublished": "2024-01-13T00:00:00",
    "assignerShortName": "mitre"
  },
  "dataVersion": "5.1"
}